src/main/java/racing/domain/Simulator.java

```diff
@@ -15,9 +16,15 @@ public void create(String names) {
         }
     }
 
+    public void move(Car car, int random) {
+        if (random > MOVE_THRESHOLD) {
+            car.move();
+        }
+    }
+
     public void run(Random random) {
         for (Car car : cars) {
-            car.move(random.nextInt(9));
+            move(car, random.nextInt(9));
```

Comment:
요구 사항의 임의 값은 0에서 9 사이입니다.

```suggestion
            move(car, random.nextInt(10));
```

Reply:
수정하였습니다.